hulearn/experimental/interactive.py

```diff
@@ -118,13 +122,14 @@ class SingleInteractiveChart:
         x: the column from the dataset to place on the x-axis
         y: the column from the dataset to place on the y-axis
         size: the size of the drawn points
-        alpha: the alpha (see-through-ness) of the drawn points
+        alpha: changes the opacity of the data points
         width: the width of the chart
         height: the height of the chart
         color: you can manually override the color of the dots to be determined by a column
           in a dataframe. This setting is useful when you want to input a list of labels but
           still want to color the dots based on a column value.
         legend: show a legend as well
+        tooltip: accepts list of the columns that will show up on hover
```

Comment:
Thanks for the PR! 

I just tried the feature and it indeed seems to work swell. The only thing I'm wondering is if it makes sense to add maybe an example in the FAQ in the docs explaining that you can do this; 

```
from hulearn.experimental.interactive import InteractiveCharts

clf = InteractiveCharts(df, labels="species")
clf.add_chart(x="bill_length_mm", y="bill_depth_mm", tooltip=df.columns)
```

That way, you don't manually need to add all the column names you'd like to see in the tooltip. 

Also, could you adapt `tests/test_interactive/test_interactive_classifier_charts.ipynb` such that it uses the tooltip feature? It's really just a smoketest but it should make it easy to spot if somebody accidentally breaks it in the future.